### PR TITLE
chore: use prettier, create lint github action

### LINKED ISSUE #42: setup Prettier + create GitHub Action that lints with Prettier?
Do we want to setup a Prettier configuration + CI that checks for Prettier errors on this repo?
I use Prettier to good effect on my simple Docusaurus v2 repo, but it might get complicated due to some autogenerating content.

Somebody who knows how the build process works should set up the `prettierignore` file so that we're only prettifying things we care about.

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Installs Prettier as a development dependency
- Create GitHub Action workflow that runs Prettier
- Fix-up current files with Prettier

## Context:

I'm trying to close #42.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: lint
+
+on:
+  pull_request:
+    branches:
+      - 'build'
+  push:
+    branches:
+      - 'build'
+      - 'renovate/**'
```
Comment (HonkingGoose) [inline .github/workflows/lint.yml:10]: Please check if I'm running the action on the correct branches!

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+env:
+  node-version: 14.x
+
+jobs:
+  build:
+    name: prettier
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
```
Comment (HonkingGoose) [inline .github/workflows/lint.yml:21]: I'm deliberately ignoring the submodule that pulls in the Renovate dependencies, as I'm ignoring it in the `.prettierignore` anyways.

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
Shouldn't this be integrated in normal build workflow?

**Comment by HonkingGoose** [comment]
> Shouldn't this be integrated in normal build workflow?

That would probably be best, but I don't really know how to do this properly. 🙈

**Pull request closed without merging**